src/codewhisperer/vue/backend.ts

```diff
@@ -156,5 +156,6 @@ export async function showCodeWhispererWebview(
                 subscriptions = undefined
             }),
         ]
+        telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_PageOpen', passive: true })
```

Comment:
The user clicked the "Learn" button, right? So why is this passive?

Reply:
This is for users for both the scenarios 
with button click and auto open so I will write a check here too

Reply:
if ((await prompts.isPromptEnabled('codeWhispererNewWelcomeMessage'))) {
            telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_PageOpen', passive: true })
        } else {
            telemetry.ui_click.emit({ elementId: 'codewhisperer_Learn_PageOpen', passive: false })
        }

        Check for both cases